content/_index.md

```diff
@@ -21,7 +21,7 @@ Remember: No question is too small.
 There are many ways you can stay involved with the Kubernetes Contributor community. 
 
 * Join a [SIG](https://git.k8s.io/community/sig-list.md)
-* Contribute to and explore our [developer guide](https://git.k8s.io/community/contributors/devel) and contributor guide
+* Contribute to and explore our [developer guide](https://git.k8s.io/community/contributors/devel) and contributor guide(https://git.k8s.io/community/contributors/guide)
```

Comment:
```suggestion
* Contribute to and explore our [developer guide](https://git.k8s.io/community/contributors/devel) and [contributor guide](/guide/)
```
If you're adding the link to the guide, I would have it refer to the version of the guide that is hosted on the site like the link above  in the [Get Involved](https://github.com/kubernetes-sigs/contributor-site/blob/master/content/_index.md#get-involved) section. 👍